post_installation_configuration/machine-configuration-tasks.adoc

```diff
@@ -26,6 +27,8 @@ link:https://access.redhat.com/solutions/4510281[updating] SSH authorized keys,
 
 {product-title} supports link:https://coreos.github.io/ignition/configuration-v3_2/[Ignition specification version 3.2]. All new machine configs you create going forward should be based on Ignition specification version 3.2. If you are upgrading your {product-title} cluster, any existing Ignition specification version 2.x machine configs will be translated automatically to specification version 3.2.
 
+There might be situations where the configuration on a node does not match fully match what the currently-applied machine config specifies. This state is called configuration drift. The Machine Config Daemon regularly checks the node configuration for configuration drift. If the MCD detects configuration drift, the MCO disables the node until an admin corrects the node configuration. For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection].
```

Comment:
I noticed this sentence used throughout the documentation updates, but I'll leave my comment in this occurrence (though it applies to all of them):

> If the MCD detects configuration drift, the MCO disables the node until an admin corrects the node configuration.

This implies (to me, anyway) that the node is no longer operational or available. When a node is degraded, it means that the machine configuration cannot be updated. In the case of config drift, it means the cluster admin must take action to restore the node to a known-good state. The node is still otherwise online and operational, with no workload disruption.

It would be more accurate to say, "the MCO marks the node degraded until..." (or similar).